classes/classes/Scenes/Areas/GlacialRift/Amarok.as

```diff
@@ -0,0 +1,164 @@
+			}
+			else if (player.getEvasionRoll()) {
+				outputText("You jump out of the way before it can hit.", false);
+				combatRoundOver();
+				return;
+			}
+			else {
+				//Damageeee + stun! Reference to the legend of it slapping a kid with its tail, except minus the bone breaking.
+				outputText("The hit sends you stumbling back", false);
+				if (player.findPerk(PerkLib.Resolute) < 0 && rand(2) == 0) 
+				{
+					outputText(", stunning you.", false);
+					player.createStatusEffect(StatusEffects.Stunned, 0, 0, 0, 0);
+				}
+				else outputText(".", false);
+				var damage:int = ((str + 100) + rand(75));
+				damage = player.reduceDamage(damage);
+				player.takeDamage(damage, true);
+				if(player.HP <= 0) {
+					doNext(game.combat.endHpLoss);
+					return;
+				}
+			}
+			combatRoundOver();
+		}
+		
+		override public function defeated(hpVictory:Boolean):void
+		{
+			game.glacialRift.amarokScene.winAgainstAmarok();
+		}
+		
+		override public function won(hpVictory:Boolean, pcCameWorms:Boolean):void
+		{
+			if(pcCameWorms){
+				outputText("\n\nThe Amarok looks down at the worms you came, sniffs them, then snarls and walks away. It must consider you tainted meat.", false);
+				doNext(game.combat.cleanupAfterCombat);
+			} else {
+				game.glacialRift.amarokScene.amarokChowTime();
+			}
+		}
+		
+		public function Amarok() 
+		{
+			this.a = "the ";
+			this.short = "Amarok";
+			this.imageName = "amarok";
+			this.long = "You are fighting an Amarok, a massive wolf that seems set on hunting you. The buffeting snow does nothing to hide its thick, black fur, and hardly manages to even ruffle it. It has golden, hungry eyes that watch your every move and sharp teeth capable of crushing bones. It looms over you in both height and width, with well-defined muscles, long legs, and bulky paws with deadly claws only adding to its intimidating stature. Even its tail looks capable of knocking you down. It's the most normal animal-like creature you've seen here yet, a normal wolf despite its size, but is no less terrifying. You get the feeling this won't be an easy fight, especially considering it's not about to let you run away.";
+			this.createCock(8, 1.5, CockTypesEnum.WOLF);
+			this.balls = 2;
+			this.ballSize = 2;
+			this.cumMultiplier = 1;
+			this.createVagina(false, 1, 1);
+			this.gender = genderless;
+			this.pronoun1 = "it";
+			this.pronoun2 = "it";
+			this.pronoun3 = "its";
+			createBreastRow();
+			createBreastRow();
+			createBreastRow();
+			this.ass.analLooseness = ANAL_LOOSENESS_TIGHT;
+			this.ass.analWetness = ANAL_WETNESS_DRY;
+			this.tallness = 8*12;
+			this.hipRating = HIP_RATING_AVERAGE;
+			this.buttRating = BUTT_RATING_AVERAGE;
+			this.lowerBody = LOWER_BODY_TYPE_WOLF;
+			this.skinTone = "black";
+			this.skinType = SKIN_TYPE_FUR;
+			this.hairColor = "black";
+			this.faceType = FACE_WOLF;
+			this.earType = EARS_WOLF;
+			initStrTouSpeInte(110, 80, 85, 95);
```

Comment:
>They have to leave their babies from time to time.

Yeah. In a company of living, dangerous creature (aka player). Very smart move.

>not even wolf-morph or biped, a literal wolf

You've listed quite a list of weirdshit, and know about much more even weirder... And still being unsettled by quadripedals? People are weird.